### ISSUE TITLE
ValueError  Missing staticfiles manifest entry for 'admin/js/jquery.min.js'

### ISSUE THREAD

**Issue by marcelhekking** [s-block/django-nested-inline#108]
This error occurred when we deployed our web application to `production`. While developing locally, there were no errors at all. Somehow, the file `jquery.min.js` was not available at `admin/js` after running collectstatic. We fixed it temporarily by overriding the NestedInline class and changing the `js` variable in the `media` method:

```Python
...
js = ['vendor/jquery/jquery%s.js' % extra, 'jquery.init.js', 'inlines%s.js' % extra]
...
```

**Comment by marcelhekking** [comment]
I see now that the root of the problem is the same as described in :  `Update media so it expects to find jquery in the right place. #75 opened on Jan 13, 2017 by lucaswiman`

**Comment by OskarPersson** [comment]
Thank you for bumping this issue 👍 , like you said it was fixed in #75 which is now merged

**Comment by rapkyt** [comment]
@OskarPersson #75 is merged but the changes are not in pity, i.e.

running `pip install django-nested-inline` would not have this change
